lib/bls.ex

```diff
@@ -77,4 +81,15 @@ defmodule Bls do
       {:error, _} -> false
     end
   end
+
+  @doc """
+  Converts private to public key
+  """
+  @spec derive_pubkey?(privkey()) :: {:ok, pubkey()} | {:error, any()}
+  def derive_pubkey?(private_key) do
```

Comment:
The `?` at the end is because of an Elixir naming convention for functions that return booleans. We don't return booleans here, so we should remove the `?`.